.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: true
+dist: trusty
+
+language: python
+
+env:
+  - KERAS_BACKEND=tensorflow
+  - KERAS_BACKEND=theano
+python:
+  - 2.7
```

Comment:
Why 2.7? Docker uses 3.6 either way.

Also, is cntk failing due to python version "leakage"?